llvm/test/CodeGen/AMDGPU/si-pre-allocate-wwm-regs.mir

```diff
@@ -0,0 +1,44 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -verify-machineinstrs -run-pass=si-pre-allocate-wwm-regs -o - -mcpu=tahiti %s  | FileCheck %s
+# RUN: llc -mtriple=amdgcn -verify-machineinstrs -amdgpu-prealloc-sgpr-spill-vgprs -run-pass=si-pre-allocate-wwm-regs -o - -mcpu=tahiti %s | FileCheck %s --check-prefix=CHECK2
+
+---
```

Comment:
Add fixme to check the MachineFunctionInfo reserved register information

Reply:
Is it for WWM reserved registers?

Reply:
Yes

Reply:
But we have that already(?)
```
wwmReservedRegs:
  - '$vgpr0'
```

Reply:
Then you need to write manual checks for it. Update_mir_test_checks doesn't currently support the function level properties

Reply:
Added a comment for the manual check.